src/ParameterDistributions.jl

```diff
@@ -452,7 +464,7 @@ Here, `xarray` contains parameters sample ensembles for different EKP iterations
 """
 function transform_unconstrained_to_constrained(
     pd::ParameterDistribution,
-    xarray::Array{Array{FT, 2}, 1},
+    xarray::AbstractVector{Matrix{FT}},
```

Comment:
I would leave this untyped as you just need to support an iterable that returns (abstract) matrices